libs/datamodel/connectors/sql-datamodel-connector/src/mysql_datamodel_connector.rs

```diff
@@ -137,6 +138,23 @@ impl Connector for MySqlDatamodelConnector {
         &self.capabilities
     }
 
+    fn validate_field(&self, field: &Field) -> Result<(), ConnectorError> {
+        if let FieldType::NativeType(_scalar_type, native_type) = field.field_type() {
+            match native_type.name.as_str() {
+                TEXT_TYPE_NAME => {
```

Comment:
I think we also need to disallow the other text types like e.g. `BigText` and so forth.

Reply:
Got it, so for `LongText`, `MediumText`, `Text` and `TinyText`, right?

Reply:
I guess so. Although `TinyText` is so small that it might be allowed actually. Would be good to check.

Reply:
just read this: https://stackoverflow.com/questions/1827063/mysql-error-key-specification-without-a-key-length/1827099#1827099

seems like all `Text` and `Blob` columns are disallowed to be unique.